srcpkgs/kde/template

```diff
@@ -1,7 +1,7 @@
-# Template file for 'kde5'
-pkgname=kde5
+# Template file for 'kde'
+pkgname=kde
```

Comment:
Should be something like `plasma` or `kde-plasma`

Reply:
there's a bunch of `plasma-*` packages, and `kdeplasma-addons`, perhaps `kdeplasma`? `kde-plasma` looks a bit better i think.